hack/deploy/controller-plugin.yaml

```diff
@@ -47,19 +47,6 @@ spec:
         volumeMounts:
         - name: socket-dir
           mountPath: /var/lib/csi/sockets/pluginproxy/
-      - name: csi-cluster
```

Comment:
Do to you need to add the CSIDriver object manually?

Reply:
Yes, I need to deploy `CSIDriver` and `CSINode` objects manually

Reply:
So, add them in the installer or inside the CSI driver?

Reply:
CSI driver added to installer